Sources/Commands/SwiftTestTool.swift

```diff
@@ -168,11 +167,12 @@ public struct SwiftTestTool {
         //     .........10.........20.........30.........40.........50.........60.........70..
         print("OVERVIEW: Build and run tests")
         print("")
-        print("USAGE: swift test [specifier] [options]")
+        print("USAGE: swift test [mode] [options]")
         print("")
-        print("SPECIFIER:")
+        print("MODES:")
         print("  -s, --specifier <test-module>.<test-case>         Run a test case subclass")
         print("  -s, --specifier <test-module>.<test-case>/<test>  Run a specific test method")
+        print("  -l, --list-tests                                  Lists test methods in specifier format")
```

Comment:
@ddunbar This is overflowing 70 columns, is it a hard limit? I can make it "Lists test methods" or move in new line?